yt_dlp/extractor/callin.py

```diff
@@ -0,0 +1,123 @@
+# coding: utf-8
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    float_or_none,
+    int_or_none
+)
+
+
+class CallinIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?callin\.com/(episode)/(?P<id>[-a-zA-Z]+)'
+    _TESTS = [{
+        'url': 'https://www.callin.com/episode/the-title-ix-regime-and-the-long-march-through-EBfXYSrsjc',
+        'info_dict': {
+            'id': '218b979630a35ead12c6fd096f2996c56c37e4d0dc1f6dc0feada32dcf7b31cd',
+            'title': 'The Title IX Regime and the Long March Through and Beyond the Institutions',
+            'url': 'https://d26nqus0kmgkkk.cloudfront.net/25246512e4b374daea3a23397483c50d241329ff8af4a5bafae02dae06cb0d5f/25246512e4b374daea3a23397483c50d241329ff8af4a5bafae02dae06cb0d5f.m3u8',
+            'ext': 'ts',
+            'display_id': 'the-title-ix-regime-and-the-long-march-through-EBfXYSrsjc',
+            'thumbnail': 'https://d2xpqol0sc5yi.cloudfront.net/82cf84f4f7b70e3e1f9e36505cce56d97a31d3bd0589156595f362d25e012e06.png',
```

Comment:
is this stable to be part of test? Otherwise, use a regex (`'re:https:\\...'`)

Reply:
It's been stable thus far; I don't know how likely it is to be stable in the medium term. Do you find that Cloudfront CDN urls often change?

Reply:
I would assume so. This should be a sufficient test.
```suggestion
            'thumbnail': 're:https://.+\.png',
```

Reply:
Done (in 6b5ec0e890b569b5d9b60e38a43f66bb1513f373)

Reply:
fyi: you don't need to link the commit for each review comment. I get notified when u make changes